waltz-ng/client/measurable-rating/components/app-section/measurable-rating-app-section.html

```diff
@@ -43,6 +43,7 @@
                      class="wt-tab">
                     <br>
                     <waltz-measurable-rating-panel ratings="tab.ratings"
+                                                   entity-reference="$ctrl.entityReference"
```

Comment:
need to pass entity reference down the stack so that urls to view page can be rendered later